spec/unit/util/dsc/local_configuration_manager_spec.rb

```diff
@@ -63,7 +63,35 @@
         let(:lcm_standard_error) { nil }
         let(:lcm_cmdlet_success) { true }
 
-        it "should successfully return resource information for normally formatted output when cmdlet the cmdlet succeeds" do
+        it "successfully return resource information for normally formatted output when cmdlet the cmdlet succeeds" do
```

Comment:
`successfully *returns* resource information for normally formatted output when the cmdlet succeeds.`